cloudinary-image-management-and-manipulation-in-the-cloud-cdn/js/src/components/taxonomies.js

```diff
@@ -0,0 +1,17 @@
+const toggler = document.querySelector( '.cloudinary-collapsible__toggle' );
+
+if ( toggler ) {
+  toggler.addEventListener( 'click', function () {
+    const content = document.querySelector( '.cloudinary-collapsible__content' );
+    const isHidden = window.getComputedStyle( content, null ).getPropertyValue( 'display' ) === 'none';
+    const arrowIcon = document.querySelector( '.cloudinary-collapsible__toggle button i' );
+  
+    content.style.display = isHidden ? 'block' : 'none'
+  
+    if ( arrowIcon.classList.contains( 'dashicons-arrow-down-alt2' ) ) {
+      arrowIcon.classList.replace( 'dashicons-arrow-down-alt2', 'dashicons-arrow-up-alt2' );
```

Comment:
@dugajean 

I have a concern here related to the usage of `classList.replace`, which isn't supported by [some browsers](https://caniuse.com/mdn-api_element_classlist_replace) — including IE 11, Edge until 16 and Safari.

However, I should note that I tested this on Safari, and it seems to be working fine.

Reply:
Thanks @pereirinha ! I will change the approach here 👍